src/systems/alias_elimination.jl

```diff
@@ -6,7 +6,10 @@ const KEEP = typemin(Int)
 
 function alias_eliminate_graph!(state::TransformationState)
     mm = linear_subsys_adjmat(state)
-    size(mm, 1) == 0 && return AliasGraph(ndsts(state.structure.graph)), mm, BitSet() # No linear subsystems
+    if size(mm, 1) == 0
+        ag = AliasGraph(ndsts(state.structure.graph))
+        return ag, ag, mm, mm, BitSet() # No linear subsystems
```

Comment:
```suggestion
        return ag, mm, ag, mm, BitSet() # No linear subsystems
```
to match the updated order of return values